docs/docs/rest-api/public/api/v2/tasks.raml

```diff
@@ -0,0 +1,43 @@
+get:
+  description:
+    List all running tasks.
+  is: [ secured ]
+  queryParameters:
+    status:
+      description: Filter the list of tasks by status
+      enum: [ TASK_STAGING, TASK_STARTING, TASK_RUNNING, TASK_FINISHED, TASK_FAILED, TASK_KILLED, TASK_LOST, TASK_ERROR ]
+      repeat: true
+  responses:
+    200:
+      description: The list of running tasks.
```

Comment:
The list of all tasks disregarding their status, or a list of tasks matching the specified `status` filter.